### DISCUSSION TITLE
小毛蟹回家 啟動台灣大轉骨｜人均GDP超韓趕日，淡水河也迎來50年生態奇蹟！

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#749] [General]
淡水河在經過這麼多年的整治後，環境終於有所改善，還代表著我們的產業由高污染且附加價值低的產業轉成低污染且附加價值高的產業，聽說毛蟹跟大閘蟹是親戚，希望我們以後都有美味的螃蟹可以吃

網址：https://www.businessweekly.com.tw/magazine/Article_page.aspx?id=6008637

**Comment by williamyen0623** [comment]
好餓 好想吃螃蟹

**Comment by Pietra1226** [comment]
感覺台灣很多地方都可做這種轉型整治。如果確實執行感覺很有潛力

**Comment by delia07152** [comment]
身為淡水人我還是覺得淡水河很髒，不過然新聞這樣說，就代表可能真的有稍微改善，希望有朝一日可以看到淡水河呈現清澈透明的顏色。

**Comment by CPCpyt** [comment]
這樣的整治的確是有必要的，
如何改善環境的同時又能促進經濟，
這是對於台灣可以思考的議題。

**Comment by ruthchen1020** [comment]
我覺得人類跟環境是互相牽制又共生的，希望我們可以把永續發展當成接下來的發展目標之一。

**Comment by StevenWanglolz** [comment]
淡水河的水質改善太好了! 不過中央附近甚麼時候會有海鮮餐廳啊? 我也好想吃螃蟹大餐喔~

**Comment by Evan0979** [comment]
希望地球的生態能夠越來越好,讓我們的子孫也能健康地活著

**Comment by peggy0329** [comment]
產業快速變遷，使得工廠也被迫搬離這個逐漸變為高科技區域的地方
當然也有政府的努力 才可以還給我們一個較乾淨的淡水河
話說好久沒去淡水了，下次可以去走走！

**Comment by swerfg03joe** [comment]
經濟發展的同時必然伴隨著許多的環境汙染，以及造成大量野生動植物的棲地消失，台灣能在這段時間的成長過後，漸漸改變表現出對環境的關懷，與時代的進步和政府環境保護法案也有相當程度的關聯，期待未來追求經濟的同時還能適當保護周遭的土地，不然最終受害的還是居住在這片土地上的我們

**Comment by WEI-CHEN-CHUN** [comment]
環境如果有改善，人民生活比較舒服何嘗不是件壞事呢?

**Comment by Kotori0501** [comment]
環境永續是真的很重要，每個人都有愛護大自然的責任存在，因此從自己的周遭環境先負責好，然後可以多參加環保活動，這是目前的方法之一。
另外，台灣確實有很多轉型突進，可以多利用再生能源，尤其是在偏海地區因為風比較大的，所以可以多多利用風力能源，我個人認為算是還不錯的選擇之一。
